docs/001_develop/03_client-capabilities/008_interaction/listbox.mdx

```diff
@@ -70,7 +72,7 @@ The primary use of this component is to be used as part of other custom form com
 
     <!-- TODO add link to sync -->
     :::warning
-    Unlike most other Genesis components you cannot use the `sync` directive here. This is because the listbox doesn't fire events, that is intended to be controlled by the parent form element.
+    Unlike most other Genesis components, you cannot use the `sync` directive here. This is because the listbox doesn't fire events. This must be controlled by the parent form element.
```

Comment:
You can action the TODO now - the section to link to is here https://docs.genesis.global/docs/develop/client-capabilities/custom-components/directives/#sync